native_client/ctcdecode/build_common.py

```diff
@@ -32,14 +32,14 @@
         'unittest.cc'))
 ]
 
-def build_common(out_name='common.a', build_dir='temp_build/temp_build', debug=False, num_parallel=1):
+def build_common(srcs=[], out_name='common.a', build_dir='temp_build/temp_build', debug=False, num_parallel=1):
```

Comment:
`build_common` is no longer a good name, and the default value for `out_name` also doesn't make sense.

Reply:
@reuben would `build_ctc.py` be fine? and `kenlm.a` for default value ?

Reply:
Just `build_archive` is fine.

Reply:
And no default value.